container_api/src/config.rs

```diff
@@ -139,14 +139,15 @@ impl TryFrom<DNAConfiguration> for Dna {
 }
 
 /// An instance combines a DNA with an agent.
-/// Each instance has its own storage and logger configuration.
+/// Each instance has its own network, storage and logger configuration.
 #[derive(Deserialize, Serialize, Clone)]
 pub struct InstanceConfiguration {
     pub id: String,
     pub dna: String,
     pub agent: String,
     pub logger: LoggerConfiguration,
     pub storage: StorageConfiguration,
+    pub network: String,
```

Comment:
```suggestion
    pub network: Option<String>,
```

Reply:
This change would make several other changes redundant.

Reply:
With this we can have a config.toml be valid even without mentioning network at all. Not even with `network=""`, which is not what I meant.